cms/tests/test_extensions.py

```diff
@@ -31,8 +31,9 @@ def test_register_extension(self):
         page_extension = self.get_page_extension_class()
 
         # register first time
-        extension_pool.register(page_extension)
+        register_extension_result = extension_pool.register(page_extension)
```

Comment:
The reasons I created the test like that are:

1. By definition decorators are functions which accept function/class as argument and returns function/class. It can return new function/class which inherit the functionality of the original or simply modify the original and return it. The syntax with `@` is syntax sugar from Python 2.6 and up. It's not required to use always a decorator with this syntax.
2. If decorate one of the Extension models in `test_utils/project/extensionapp/models.py` with @-syntax and the decorator functionality is broken (`register` does not return class at all) then when tests are executed, following error will be raised `TypeError: 'NoneType' object is not iterable` even before the tests can be started. This is not useful for developer who accidentally broke that functionality.

If you still prefer one (or both) of the extension models to be decorated with @-syntax in the test app and to revert the change in `test_extensions.py`,  I will do it.